init/docker/.env.example

```diff
@@ -0,0 +1,21 @@
+INFLUXDB_USERNAME=admin
+INFLUXDB_PASSWORD=admin
+INFLUXDB_DATABASE=unifi
+
+GRAFANA_USERNAME=admin
+GRAFANA_PASSWORD=admin
+GRAFANA_PLUGINS=grafana-clock-panel,natel-discrete-panel,grafana-piechart-panel
+
+UP_POLLING_MODE=influx
```

Comment:
I'll note here for now that this (env vars) does not work with the current release. It will work in the next release.

Reply:
Is it only that specific env var? Or all of them?

Reply:
None of these variables are in 1.5.2. They are in the beta files I've been passing out on Discord that also say 1.5.2, but these have not made it to an official release yet. They'll be in the next one.

Reply:
In other words, `stable` does not work with this compose file.

Reply:
Luckily you can specify the branch with UP_BRANCH in the .env file :)